packages/vaex-ml/vaex/ml/transformations.py

```diff
@@ -127,20 +122,14 @@ def transform(self, df, n_components=None):
         for i in range(n_components):
             vector = eigen_vectors[:, i]
             if self.whiten:
```

Comment:
a slightly cleaner way to do this could be 
```
expr = copy.func.dot_product(expressions, vector)
if self.whiten:
    expr = f'({expr}) / {np.sqrt(self.explained_variance_[i])}'
```

Reply:
I agree, do you want to change that Jovan, ok are you ok with this?

Reply:
Yeah I agree